google-cloud-examples/src/main/java/com/google/cloud/examples/bigtable/HelloWorld.java

```diff
@@ -85,9 +98,10 @@ public void run() throws Exception {
     adminClient.close();
   }
 
+  /** Demonstrates how to create a table. */
   public void createTable() {
     // [START creating_a_table]
-    // Check if table exists, create table if does not exist
+    // Checks if table exists, creates table if does not exist.
```

Comment:
It would read better if we rephrase it to: " Checks if table exists, creates a table if it does not exist."